packages/js/components/src/index.ts

```diff
@@ -32,7 +32,7 @@ export { default as Pill } from './pill';
 export { default as Plugins } from './plugins';
 export { default as ProductImage } from './product-image';
 export { default as ProductRating } from './rating/product';
-export { default as Rating } from './rating';
+export { default as Rating } from './rating/index';
```

Comment:
Seems a bit unusual to have ./index, is this required?

Reply:
Thank you for catching it 🙏 Updated in https://github.com/woocommerce/woocommerce/pull/36301/commits/947e3b76fb551cc112e3cb909019cf47706d9364